### PR TITLE
html-form

### PR DESCRIPTION
**Pull request by AndresR9707**

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="es">
+    <head>
+    <meta charset="utf-8">
+    <meta name="robots" content="2">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="og:title" property="og:title" content="Ferreteria Trinidad la 14">
+    <meta name="description" content="La ferretería Trinidad ofrece los servicios de cerrajero y plomero a domicilio en la ciudad de tunja, además encuentra en nuestras instalaciones todos los materiales para la construcción">
+    <title>Ferreteria Trinidad</title>
```
Comment (fabianmarinog) [inline form.html:9]: the order doesn't matter but for readability the `<title>` comes first.

File: form.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      <label for="method"><strong>Medios de pago:</strong>
+      <abbr title="required" aria-label="required">*</abbr><br>   
+       Efectivo:<input type="radio" id="cash" name="method" required><br> 
+       Tarjeta:<input type="radio" id="card" name="method" required><br>
+       PSE:<input type="radio" id="pse" name="method" required><br>
+      </label>
+    </fieldset>
+    <p>
+    <button id="btn_submit">Enviar pedido</button> 
+    <button type="reset" id="btn_reset">Eliminar</button>
```
Comment (fabianmarinog) [inline form.html:75]: this should be 
```suggestion
    <button type="reset" id="btn_reset">Cancelar</button>   
```

File: form.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    <button id="btn_submit">Enviar pedido</button> 
+    <button type="reset" id="btn_reset">Eliminar</button>   
+    </p> 
+    </form>
+    </section>
+    </article>
+    </main>
+    <footer>   
+    <hr> 
+    <label id="information">Mayor Informacion:</label>
```
Comment (fabianmarinog) [inline form.html:83]: `<label>` is more common within `<form>` and `<input>` Use `<p>`
```suggestion
    <p>Mayor Informacion:</p>
```

File: form.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+     <nav>
+      <ul>
+      <li>Telefono: 311 998 6580</li>
+      <li>Email: ferreteriatrinidad74@gmail.com</li>
+      <li>Cr 14 # 1 A sur Tunja(Boyaca)</li>
```
Comment (fabianmarinog) [inline form.html:88]:
```suggestion
      <li><address>Cr 14 # 1 A sur Tunja(Boyaca)</address></li>
```

**Review by fabianmarinog — CHANGES_REQUESTED** [review]
Good job. The general exercise is correctly implemented, just check these comments.

File: form.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        </fieldset>
+        <label for="message">Otro tipo de producto?(optional):</label><br>
+        <textarea id="message" name="message" maxlength="50" minlength="5" rows="5"></textarea>
+     <fieldset>
+      <label for="method"><strong>Medios de pago:</strong>
+      <abbr title="required" aria-label="required">*</abbr><br>   
+       Efectivo:<input type="radio" id="cash" name="method" required><br> 
+       Tarjeta:<input type="radio" id="card" name="method" required><br>
+       PSE:<input type="radio" id="pse" name="method" required><br>
+      </label>
```
Comment (fabianmarinog) [inline form.html:71]:
```suggestion
 <strong>Medios de pago:</strong>
      <abbr title="required" aria-label="required">*</abbr><br>   
        <label for="cash">Efectivo:<input type="radio" id="cash" name="method" required></label><br> 
       <label for="card">Tarjeta:<input type="radio" id="card" name="method" required></label><br>
       <label for="pse">PSE:<input type="radio" id="pse" name="method" required></label><br>
```

**Review by fabianmarinog — CHANGES_REQUESTED** [review]

File: form.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      <h2>Promociones:</h2>
+      <figure>
+      <img src="3-semantic-html/imagenes/F1.jpeg" width="250" height="250" alt="Productos de ferreteria"><br>
+      <figcaption>Productos de ferreteria</figcaption>
+      </figure>
+     </header>
+     <section>
+      <p>Esta ferreteria es un establecimiento comercial que se dedica  a la venta al por mayor y menor de materiales para construccion y necesidades del hogar</p>  
+      <img src="3-semantic-html/imagenes/F2.jpeg" width="400" height="200" alt="Fachada"><br>
+      <p>Realiza tus pedidos:</p>
```
Comment (pushdev-code) [inline form.html:37]:
```suggestion
      <h3>Realiza tus pedidos:</h3>
```
The section needs a heading and this one is the best to meet that requirement.

**Review by pushdev-code — CHANGES_REQUESTED** [review]
just one final comment

**Pull request merged**